src/components/CenteredBox.tsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+
+interface CenteredBoxProps {
+  children: React.ReactNode;
+}
+
```

Comment:
[nits]
コンポーネントのpropsは殆どの場合以下のようにされています（慣習的に）

```suggestion
type Props = {
  children: React.ReactNode;
}

```